js/script.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+/* Это 3.14ЗДЕЦ какой-то, не думал, что будет так сложно!
+
+   Про то, что пользоваться innerHTML нежелательно я помню,
+   но пока не знаю как это сделать, чтобы код не превращался в портянку
+   на пяток страниц. Дополнительные задания не делал, ибо мне и 
+   основного хватило для закипания мозга. */
+
+
+
+// Массив с данными о комментариях
+const commentsListArray = [
+    {
+        name: 'Глеб Фокин',
+        date: '12.02.22 12:18',
+        text: 'Это будет первый комментарий на этой странице',
+        likeCount: 3,
+        liked: false,
+    },
+    {
+        name: 'Варвара Н.',
+        date: '13.02.22 19:22',
+        text: 'Мне нравится как оформлена эта страница! ❤',
+        likeCount: 75,
+        liked: true,
+    },
+];
+
+// UL of comments
+const commentsList = document.querySelector('ul.comments'); 
+
+// Функция рисует HTML-разметку всех комментариев
+function renderComments() {
+    commentsList.innerHTML = commentsListArray.reduce((result, comment, index) => {
+        return result + `
+    <li class="comment">
+        <div class="comment-header">
+        <div>${comment.name}</div>
+        <div>
+            ${comment.date}
+        </div >
+        </div >
+        <div class="comment-body">
+        <div class="comment-text">
+            ${comment.text}
+        </div>
+        </div>
+        <div class="comment-footer">
+        <div class="likes">
+            <span class="likes-counter">${comment.likeCount}</span>
+            <button data-index="${index}"  class="${(comment.liked) ? 'like-button -active-like' : 'like-button'}"></button>
```

Comment:
Лайк за тернарник, он нам еще не один раз пригодится в похожих ситуациях :)

Скобки можно было не ставить, чтоб не шуметь в коде дополнительно:

```suggestion
            <button data-index="${index}"  class="${comment.liked ? 'like-button -active-like' : 'like-button'}"></button>
```

Reply:
Исправлю, спасибо